src/submit-modal/success-content.js

```diff
@@ -81,14 +83,67 @@ const copyTableToClipboard = ({ importSummaries, requests }) => {
     navigator.clipboard.writeText(clipboardText)
 }
 
-const SummaryTable = ({ importSummaries }) => {
+const SummaryRow = ({
+    importSummary,
+    exchangeName,
+    expandedRows,
+    expandToggle,
+    hasConflicts,
+}) => {
+    return importSummary.conflicts?.length > 0 ? (
+        <DataTableRow
+            expanded={expandedRows.includes(exchangeName)}
+            onExpandToggle={() => expandToggle(exchangeName)}
+            expandableContent={
+                <ConflictsDetailsTable conflicts={importSummary.conflicts} />
+            }
+        >
+            <DataTableCell>{exchangeName}</DataTableCell>
+            <DataTableCell>
+                {importSummary?.importCount?.imported}
+            </DataTableCell>
+            <DataTableCell>{importSummary?.importCount?.updated}</DataTableCell>
+            <DataTableCell>{importSummary?.importCount?.ignored}</DataTableCell>
+        </DataTableRow>
+    ) : (
+        <DataTableRow>
+            {hasConflicts && <DataTableCell></DataTableCell>}
+            <DataTableCell>{exchangeName}</DataTableCell>
+            <DataTableCell>
+                {importSummary?.importCount?.imported}
+            </DataTableCell>
+            <DataTableCell>{importSummary?.importCount?.updated}</DataTableCell>
+            <DataTableCell>{importSummary?.importCount?.ignored}</DataTableCell>
+        </DataTableRow>
+    )
+}
```

Comment:
I guess you _could_ avoid repeating yourself this way?
```suggestion
    const rowHasConflicts = importSummary.conflicts?.length > 0
    const showPaddingCell = hasConflicts && !rowHasConflicts

    return (
        <DataTableRow
            expanded={expandedRows.includes(exchangeName)}
            onExpandToggle={
                rowHasConflicts ? () => expandToggle(exchangeName) : undefined
            }
            expandableContent={
                rowHasConflicts ? (
                    <ConflictsDetailsTable
                        conflicts={importSummary.conflicts}
                    />
                ) : undefined
            }
        >
            {showPaddingCell && <DataTableCell></DataTableCell>}
            <DataTableCell>{exchangeName}</DataTableCell>
            <DataTableCell>
                {importSummary?.importCount?.imported}
            </DataTableCell>
            <DataTableCell>{importSummary?.importCount?.updated}</DataTableCell>
            <DataTableCell>{importSummary?.importCount?.ignored}</DataTableCell>
        </DataTableRow>
    )
```